apps/site/src/app/(home)/sections/Sponsors/Sponsors.module.scss

```diff
@@ -8,61 +8,68 @@
 
 .title {
 	text-align: center;
-}
-
-.clipboard {
-	position: relative;
-	padding: 64px 32px 48px 32px;
-	min-height: 512px;
-	width: 80%;
-	margin: 0 auto;
-	border: 20px solid #aa703c;
-	border-radius: 100px;
-	background: #ffffff;
-	display: flex;
-	flex-direction: column;
-	align-items: stretch;
+	color: theme.$white;
+	font-size: 3rem;
+	font-weight: bold;
+	margin-bottom: 3rem;
 
-	@include utils.media-breakpoint-up(sm) {
-		padding: 32px 32px 48px 32px;
+	@include utils.media-breakpoint-down(sm) {
+		font-size: 2rem;
 	}
```

Comment:
Suggestion [repeating]: Bootstrap recommends a "mobile-first" design approach, so making the narrower view the default and using `media-breakpoint-up` would be more appropriate.